README.md

```diff
@@ -28,30 +29,43 @@ To install vCluster Rancher Operator, you can add the loft repository to Rancher
 7. Follow the installation process and install the chart.
 8. In the sidebar, navigate to "Workloads" -> "Deployments". Confirm that the deployment named "vcluster-rancher-op" has the State "Active".
 
-Once the operator is installed, all vClusters deployed in any downstream cluster in rancher will cause a corresponding Rancher cluster to be created, the vCluster to connect to the corresponding Rancher cluster, and cluster owners added.
+Once the operator is installed:
+* All vClusters deployed in any downstream cluster will cause a corresponding Rancher cluster to be created
+* The vCluster will connect to the corresponding Rancher cluster
+* Any project-member or project-owner will be added as a cluster-owner
 
 ## Uninstall
+
 1. Select the local cluster in the Rancher clusters overview page.
 2. In the sidebar, navigate to "Apps" -> "Installed Apps".
-3. Delete the vcluster-rancher-op app. The app will be have the name you gave it during install.
+3. Delete the vcluster-rancher-op app.
+
+Note that Rancher clusters will not be affected by removing this operator, outside of operations it manages such as RBAC syncing.
 
 ## Technical Details
 
-The vCluster Rancher operator run in the local cluster and watches clusters.management.cattle.io. When a cluster has not been seen by the handler before,
-it creates a client using the Rancher reverse proxy to talk to the downstream cluster. This is done by creating a token, if needed, for the user.management.cattle.io
-that is created during the Helm chart install. The token is then used to authenticate requests to the Rancher proxy.
 
-Once a client is created for a downstream cluster, it is used to watch all services with the label `app=vcluster`. If the vCluster install does not have a corresponding
-Rancher cluster, the installation process begins. The installation process creates a provisioning clusters, waits for Rancher to create the clusterregistrationtoken.cattle.io,
-and then extracts the command from the clusterregistrationtokens status. This command contains everything that is needed to deploy Rancher's cluster agent to the vCluster. A
-job is then deployed in the vCluster's host cluster that creates a kubeconfig pointing the the vCluster service in the host cluster. The earlier extracted command is executed
-using the newly created kubeconfig.
+*Downstream Client*
+
+The vCluster Rancher operator runs in Rancher's local cluster and watches clusters.management.cattle.io. 
+When a cluster has not been seen by the handler before, it creates a client with a token, using the Rancher reverse proxy to talk to the downstream cluster.
```

Comment:
This is still inaccurate. An important part of how the operator works is that on start up it creates a token for the users that the chart installs. In then uses that going forward instead of the service account. Most of the operator is leveraging a user that has the admin global role. I think this is important context for devs and maybe even users.